common/constants.go

```diff
@@ -180,6 +180,10 @@ const (
 	VcapServiceEnvName = "VCAP_SERVICES"
 	// VcapApplicationEnvName refers to the VCAP_APPLICATION environment variable DefaultDockerfileName
 	VcapApplicationEnvName = "VCAP_APPLICATION"
+	// VcapSpringBootSecretSuffix refers to VCAP springboot secret suffix
+	VcapSpringBootSecretSuffix = "-vcapspringboot"
+	// VcapCfSecretSuffix refers to VCAP secret suffix
+	VcapCfSecretSuffix = "-vcapcf"
```

Comment:
can we call it something like say, -vcapasspringbootproperties and -vcapasenv? also possibly change the description to state what they actually store.

Reply:
done